cdap-ui/app/features/tracker/templates/metadata.html

```diff
@@ -25,48 +25,81 @@
   <!-- TAGS SIDEBAR -->
   <div class="entity-detail-tags-column">
     <div class="entity-container">
-      <div class="user-tags" ng-if="MetadataController.tags.user.length > 0">
+      <div class="user-tags" ng-if="MetadataController.tags.userTags.length > 0 || MetadataController.tags.preferredTags.length > 0">
```

Comment:
this condition is wrong. If there is no userTags or preferredTags, it should still show the plus button